examples/unconditional-image-generation/README.md

```diff
@@ -0,0 +1,38 @@
+<!---
+Copyright 2024 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
```

Comment:
Can you move this whole example script and Readme under stable_diffusion? You can create unconditional_image_generation.py.

Reply:
Hi,
Moved the example under stable_diffusion and combined the readme